### DISCUSSION TITLE
Help with gen-ref, init_InhoCorrection and despike errors

### DISCUSSION THREAD

**Discussion by HelloKitty4lif3** [CoBrALab/RABIES#360] [Q&A]
Hi

I am running preprocessing with this data set from OpenNeuro: https://openneuro.org/datasets/ds004114/versions/1.0.0 

This is command I am running: 
![image](https://github.com/CoBrALab/RABIES/assets/161685735/6167a7e0-029f-42fd-bc02-9937db12fa67)

My compile creates errors and I receive different types of crash files which I would like to improve on. I tried running the same code but instead on the test dataset provided by rabies. I get the same types of crash errors. The crash files seem to be "gen-ref", "init-InhoCorrection" and "despike". I have looked through the rabies documentation but without luck. Have anybody here faced the same types of crash messages and what were your solution to them?

I have attached my preprocess logs of my runs with the OpenNeuro dataset and the test dataset.
[rabies_preprocess_dataset.log](https://github.com/CoBrALab/RABIES/files/14495685/rabies_preprocess_dataset.log)
[testdataset_rabies_preprocess.log](https://github.com/CoBrALab/RABIES/files/14495674/testdataset_rabies_preprocess.log)

**Comment by gdevenyi** [comment]
How did you install and run rabies. The supported method is the singularity container.

**Reply by HelloKitty4lif3** [reply]
I have anaconda3 environment installed on a CentOS server where I run rabies.

**Reply by gdevenyi** [reply]
We cannot debug pipeline failures in an uncontrolled installation environment. Please reproduce your issue using the supplied docker or singularity container runtimes.
